mlrun/launcher/local.py

```diff
@@ -22,13 +21,11 @@
 import mlrun.runtimes.generators
 import mlrun.utils.clones
 import mlrun.utils.notifications
-from mlrun.launcher.base import BaseLauncher
+from mlrun.launcher.client import ClientBaseLauncher
```

Comment:
seems like we have a mix of imports of the other Launchers.
In local.py and remote.py we import with `from x` and on the `client.py` we import with `import x`, I suggest we be aligned and use the `import x` instead of `from`
```suggestion
import mlrun.launcher.client
```